app/lib/features/chat/widgets/custom_input.dart

```diff
@@ -903,35 +906,9 @@ class _TextInputWidgetConsumerState extends ConsumerState<_TextInputWidget> {
             ),
             child: const Icon(Icons.emoji_emotions),
           ),
-          border: OutlineInputBorder(
```

Comment:
Not convinced by removing this design. See UX comment.

Reply:
Will do the needful.